docs/site/index.md

```diff
@@ -20,6 +20,29 @@ multiple repos, as in v3. However, examples and externally-developed components
 will be in separate repositories.
 '%}
 
+LoopBack is a highly extensible, open-source Node.js framework based on Express
+that enables you to quickly create dynamic end-to-end REST APIs and connect to
```

Comment:
`create dynamic end-to-end REST APIs and connect to` => create APIs and Microservices composed from?